src/Components/Navigation.js

```diff
@@ -153,4 +142,37 @@ class Navigation extends React.Component {
   }
 }
 
+function ActualNavigation({
```

Comment:
Sorry, but I meant the following order of components:

1.) `ActualNavigation`
2.) `BackgroundVideo`
3.) `Navigation`

Can you move `ActualNavigation` and `BackgroundVideo` to the top of the file?

I probably did not make that clear in https://github.com/Scrivito/scrivito_example_app_js/pull/121#discussion_r218410285.